### DISCUSSION TITLE
Export model to gltf format via API

### DISCUSSION THREAD

**Discussion by vitaly-novikov** [Dlubal-Software/Dlubal_CSharp_Client#63] [Q&A] [answered]
Is it possible to programmatically (via API) export a model in gltf (gltb) format in rfem 5.29? If not, is it possible in IFC format?

**Comment by uncracked-concrete** [comment, marked as answer]
As per documentation of RFEM 5, it's not possible to export the model to any format (except saving it).
But as this is the repo of the RFEM 6 API: If you can use RFEM 6, there is a `export_to_ifc` function.
